### DISCUSSION TITLE
markdownの$..$数式展開がされない場合がある

### DISCUSSION THREAD

**Discussion by gyu-don** [increments/qiita-discussions#189] [要望・アイデア・不具合]
たとえば
`あああ$\tilde{a}_{X}\tilde{a}_{X}$いいい`
がインラインで数式展開されません。

![Screenshot_20220327_172437](https://user-images.githubusercontent.com/13610313/160273302-2b685caa-8879-4696-97bb-5e76b1c4aa08.png)

不具合かと思いましたが、「書き方間違ってるよ」や、とりあえずの回避法があれば教えていただけると助かります。

スクショにある検証は、こちらの記事で行いました。
https://qiita.com/gyu-don/private/a149f07922d5b4b7c20a

**Comment by gyu-don** [comment]
検証記事に追記しましたが、Qiitaに限らない既知の事象で、workaroundとして `_` の前後に半角スペースを入れるなどがあることを教えていただきました。
https://kmyk.github.io/blog/blog/2020/02/19/portable-mathjax-markdown/

**Comment by kskwtnk** [comment]
@gyu-don 
ご意見＆検証ありがとうございます！

Markdown記法の`強調`と書き方がかぶってしまっているのと、回避するには数式かどうかを判別してパースする必要があるので対応が難しそうです 🙇 
@gyu-don さんがコメントしてくださっているような回避策をとっていただくのが現状ベストに思えるため、お手数をおかけしますが引き続きこちらの書き方でお願いします 🙇

**Reply by gyu-don** [reply]
承知いたしました！ありがとうございます！
